pkg/tbtc/node.go

```diff
@@ -1,17 +1,222 @@
 package tbtc
 
 import (
+	"bytes"
 	"fmt"
 	"math/big"
+	"sort"
 
 	"github.com/keep-network/keep-common/pkg/persistence"
+	"github.com/keep-network/keep-core/pkg/chain"
 	"github.com/keep-network/keep-core/pkg/generator"
 	"github.com/keep-network/keep-core/pkg/internal/testutils"
 	"github.com/keep-network/keep-core/pkg/net"
 	"github.com/keep-network/keep-core/pkg/protocol/group"
 	"github.com/keep-network/keep-core/pkg/tecdsa/dkg"
 )
 
+// dkgResultSigner is responsible for signing the DKG result and verification of
+//  signatures generated by other group members.
+type dkgResultSigner struct {
```

Comment:
The `dkgResultSigner` and `dkgResultSubmitter` look very well! Few comments about them:
- Let's place them at the bottom of the file. The main and most important type of this file is `node` so it would be good to keep it at the top. 
- Let's remove the redundant space from the `dkgResultSigner` docstring.
- Let's add unit tests that will cover both types' methods

Reply:
Done in b4c46c4b232dc2ac8c715ecdec51f38b60400f3b, except for the unit tests.

As discussed, I added a TODO above every function in `dkgResultSigner` and `dkgResultSubmitter` to write unit tests.
They can be added in a separate PR.